### PR TITLE
NVMe baremetal test cluster config and suite

### PR DESCRIPTION
**Pull request by sunilkumarn417**
# Description

- NVMe baremetal test cluster config and suite
- 12 node cluster configuration
- Test suite has customzied OSD deployment and a sanity test to ensure readiness for next suites.

### CODE REVIEW

**Review by rahullepakshi — DISMISSED** [review]

**Review by HaruChebrolu — APPROVED** [review]

File: suites/reef/baremetal/deploy/rh/nvme_cali_1admin_6node_2client.yaml [hunk 1]
```diff
@@ -0,0 +1,164 @@
+#===============================================================================================
+# Suite: clara_suite.yaml
```
Comment (HaruChebrolu) [inline suites/reef/baremetal/deploy/rh/nvme_cali_1admin_6node_2client.yaml:2]: I think we need to change this.

**Review by tintumathew10 — COMMENTED** [review]
LGTM

File: suites/reef/baremetal/deploy/rh/nvme_cali_1admin_6node_2client.yaml (hunk 1 above)
Comment (sunilkumarn417) [reply suites/reef/baremetal/deploy/rh/nvme_cali_1admin_6node_2client.yaml]: Done

**Review by tintumathew10 — COMMENTED** [review]
LGTM

**Review by rahullepakshi — APPROVED** [review]
LGTM

**Review by HaruChebrolu — APPROVED** [review]
LGTM

**Review by manasagowri — APPROVED** [review]

**Pull request merged**